src/main/java/net/blay09/mods/prettybeaches/PrettyBeachesConfig.java

```diff
@@ -16,6 +16,10 @@
     @Config.Name("Animated Flooding")
     @Config.Comment("Whether the flooding of adjacent air blocks should be animated or instant.")
     public static boolean animatedFlooding = true;
+    
+    @Config.Name("Ignore Broken Block")
+    @Config.Comment("Whether the destroyed block gets a check that if it's in Affected Blocks list or not.")
+    public static boolean ignoreBrokenBlock = false;
```

Comment:
Would prefer if affectedBlocks could support a simple wildcard ("*") instead of a new separate option, something like this:

```java
public static boolean isBlockAffected(Block block) {
    return Arrays.contains(affectedBlocks, "*") || affectedBlocksList.contains(block);
}
```